src/edge_kernels/ContinuityEdgeSolverAlg.C

```diff
@@ -56,12 +55,20 @@ ContinuityEdgeSolverAlg::execute()
   // STK stk::mesh::NgpField instances for capture by lambda
   const auto& fieldMgr = realm_.ngp_field_manager();
   const auto coordinates = fieldMgr.get_field<double>(coordinates_);
-  const auto velocity = fieldMgr.get_field<double>(velocityRTM_);
+  const auto velocity = fieldMgr.get_field<double>(velocity_);
```

Comment:
Why was this changed to velocity? Does this make sense for rigid body motion when the GCL computation isn't needed i.e. rotation overset cylinder?